src/Management/src/EndpointCore/SpringBootAdminClient/Application.cs

```diff
@@ -22,15 +23,7 @@ internal class Application
         public Uri ServiceUrl { get; set; }
 
         [JsonPropertyName("metadata")]
-        public Metadata Metadata { get; set; }
-    }
-
-#pragma warning disable SA1402 // File may only contain a single type
-    internal class Metadata
-#pragma warning restore SA1402 // File may only contain a single type
-    {
-        [JsonPropertyName("startup")]
-        public DateTimeOffset Startup { get; set; }
+        public Dictionary<string, object> Metadata { get; set; }
     }
 
 #pragma warning disable SA1402 // File may only contain a single type
```

Comment:
SA1402 does not need to be disabled anymore

Reply:
SA1402 is in many other places. Let me make a separate commit for this instead of mixing this in.